service-workers/service-worker/multi-globals/relevant/test-sw.js

```diff
@@ -0,0 +1,5 @@
+this.addEventListener('fetch', event => {
```

Comment:
The second snippet in https://github.com/w3c/web-platform-tests/pull/3449/commits/4b90437e8c2a8cca4b8c4af28d1893ac6f0fc61c#r100237660 needs to be added in this script and to other service worker scripts I guess.